static/js/apps/eval_rig/eval_section.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GoogleSpreadsheet } from "google-spreadsheet";
+import React, { useEffect, useState } from "react";
+
+export interface EvalSectionProps {
+  doc: GoogleSpreadsheet;
+  rowIdx: number;
+}
+
+export function EvalSection(props: EvalSectionProps): JSX.Element {
+  const [dcQuestion, setDcQuestion] = useState<string>("");
+  const [dcAnswer, setDcAnswer] = useState<string>("");
+  const [llmStat, setLlmStat] = useState<string>("");
+  const [dcStat, setDcStat] = useState<string>("");
+
+  useEffect(() => {
+    const callSheet = props.doc.sheetsByTitle["dc_calls"];
+    const filter = [];
+    filter.push(`C${props.rowIdx + 1}:F${props.rowIdx + 1}`);
```

Comment:
For later...

Referring to column indexes is a bit brittle, do you think in loadQuery and loadCall we can store the mapping from column name => column idx, and use that map here?

Reply:
Updated to load all sheet headers and use that to get column index. It does feel cleaner now.